Template and VF Components/TestQuery.cls

```diff
@@ -0,0 +1,44 @@
+@isTest
+public class TestQuery 
+{
+    @testsetup
+    static void TestQuery()
+    {
+        List<contact> newClasses = new List<contact>();
```

Comment:
variable name is `newClasses` but the data type is `List<contact>` 
doesn't make any sense

Reply:
I changed it